### ISSUE TITLE
add a share on linkedin

### ISSUE THREAD

**Issue by piamancini** [opencollective/backyourstack#32]
> - Add share to LinkedIN, I know it's weird but I can think of companies sharing that on LinkedIN

**Comment by cuiki** [comment]
The mocks reflect the Share on Linked button as well. You meant to add the button along the other two buttons right? Tweet and FB Share.

**Comment by piamancini** [comment]
yeah, but we'll do it for a later v. it's not so important and takes up a lot of space
